grabbit/src/main/groovy/com/twcable/grabbit/spring/batch/repository/servlets/GrabbitCleanJobRepositoryServlet.groovy

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015 Time Warner Cable, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twcable.grabbit.spring.batch.repository.servlets
+
+import com.twcable.grabbit.spring.batch.repository.services.CleanJobRepository
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+import org.apache.felix.scr.annotations.Reference
+import org.apache.felix.scr.annotations.sling.SlingServlet
+import org.apache.sling.api.SlingHttpServletRequest
+import org.apache.sling.api.SlingHttpServletResponse
+import org.apache.sling.api.servlets.SlingAllMethodsServlet
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet
+
+import javax.annotation.Nonnull
+import javax.servlet.http.HttpServletResponse
+
+@Slf4j
+@CompileStatic
+@SlingServlet(methods = ['POST'], resourceTypes = ['twcable:grabbit/jobrepository/clean'])
+class GrabbitCleanJobRepositoryServlet extends SlingAllMethodsServlet {
```

Comment:
@jbornemann @jazeren1 @jdigger @masroormohammed added this. 

`POST /grabbit/jobrepository/clean with param 'hours'`

I like using a POST API to clean the jobrepository .. that acts as a "resource" .. and that resource manifests with the help of given "hours" param. This servlet returns JobExecutionIds that were removed (if successful)